app/controllers/DraftMetadataUploadController.scala

```diff
@@ -49,15 +49,16 @@ class DraftMetadataUploadController @Inject() (
       val successPage = routes.DraftMetadataChecksController.draftMetadataChecksPage(consignmentId)
       val token = request.asInstanceOf[Request[AnyContent]].token
       val uploadBucket = s"${applicationConfig.draft_metadata_s3_bucket_name}"
-      val uploadKey = s"$consignmentId/draft-metadata.csv"
+      val uploadFileName = "draft-metadata.csv"
+      val uploadKey = s"$consignmentId/$uploadFileName"
```

Comment:
I think it is best to add a property in `application.base.conf` file and use it here.

```
draftMetadata {
    fileName = "draft-metadata.csv"
}
```